config/routes.rb

```diff
@@ -21,4 +21,8 @@
 
   get '/shelters/:shelter_id/reviews/new', to: 'reviews#new'
   post '/shelters/:shelter_id', to: 'reviews#create'
+  get '/shelters/:review_id/reviews/edit', to: 'reviews#edit'
+  patch '/reviews/:review_id', to: 'reviews#update'
+  delete '/reviews/:review_id', to: 'reviews#destroy'
```

Comment:
Do we need to revise/refactor rest of the routes for shelter_id or review_id? I kind of started getting confused.

Reply:
I don't think so, but we can revisit if things aren't working.